src/components/navBar/navBar.js

```diff
@@ -209,9 +209,9 @@ MdNavBarController.prototype._initTabs = function() {
 MdNavBarController.prototype._updateTabs = function(newValue, oldValue) {
   var tabs = this._getTabs();
 
-  var oldIndex;
+  var oldIndex, oldTab;
   if (oldValue) {
```

Comment:
We can also delete this `if(oldValue)` because two lines below there is a validation `if(oldTab)` and `oldTab` will be never used again.

@gmoothart, what do you think?